docs/docs-next/docs/guides/automation.md

```diff
@@ -21,117 +21,59 @@ Automation is key to building reliable, efficient data pipelines. This guide cov
 
 Before continuing, you should be familiar with:
 
-- [Asset definitions](concepts/assets)
-- [Jobs](concepts/jobs)
+- [Asset definitions](/concepts/assets)
+- [Jobs](/concepts/ops-jobs)
 
 </details>
 
 
-
-
 ## Automation Methods Overview
 
 Dagster offers several ways to automate pipeline execution:
 
-1. Schedules - Run jobs at specified times
-2. Sensors - Trigger runs based on events
-3. Declarative Automation - Automatically materialize assets based on conditions  
-4. Asset Sensors - Trigger jobs when specific assets materialize
+1. [Schedules](#schedules) - Run jobs at specified times
+2. [Sensors](#sensors) - Trigger runs based on events
+3. [Asset Sensors](#asset-sensors) - Trigger jobs when specific assets materialize
 
 Let's look at each method in more detail.
 
 ## Schedules 
 
 Schedules allow you to run jobs at specified times, like "every Monday at 9 AM" or "daily at midnight."
-A schedule combines a selection of assets, known as a [Job](/concepts/jobs), and a cron expression in order to define when the job should be run.
+A schedule combines a selection of assets, known as a [Job](/concepts/ops-jobs), and a [cron expression](https://en.wikipedia.org/wiki/Cron) 
+in order to define when the job should be run.
+
 To make creating cron expressions easier, you can use an online tool like [Crontab Guru](https://crontab.guru/).
 
 ### When to use Schedules
 
 - You need to run jobs at regular intervals
-- You want a basic time-based automation method
-
-### Basic Schedule Example
-
-```python
-from dagster import ScheduleDefinition, define_asset_job
+- You want basic time-based automation
 
-# A job is a selection of assets that are grouped together for execution
-daily_refresh_job = define_asset_job("daily_refresh", selection=["customer_data", "sales_report"])
+For examples of how to create schedules, see the [How-To Use Schedules](/guides/automation/schedules) guide.
 
-# Create a schedule that runs the job daily at midnight
-daily_schedule = ScheduleDefinition(
-    job=daily_refresh_job,
-    cron_schedule="0 0 * * *"  # Runs at midnight daily
-)
-```
-
-View more detailed examples of schedules in the [How-To Use Schedules](/guides/automation/schedules) 
-and read more about how Schedules work in [About Schedules](/concepts/schedules).
+For more information about how Schedules work, see the [About Schedules](/concepts/schedules) concept page.
 
 ## Sensors
 
 Sensors allow you to trigger runs based on events or conditions, like a new file arriving or an external system status change.
 
-A sensor requires that you define a function that will 
+Like schedules, sensors operate on a selection of assets, known as [Jobs](/concepts/ops-jobs) and can either start a pipeline 
+through a Run or log a reason for not starting a pipeline using a SkipReason.
+
+However, unlike schedules, sensors are triggered by events that you define.
+You must provide a function that the sensor will use to determine if it should trigger a run.
```

Comment:
Those linebreaks are just my editor so the lines don't look too long. In Markdown, they don't appear as new lines when rendered. 

`Run` is a Dagster Concept, but also a general concept. It could go either way but I think lowercase looks better here. 

Agree on swapping, will update.